src/components/common/forms/ContactForm.tsx

```diff
@@ -1,28 +1,47 @@
 import React from "react";
 import { useForm } from "react-hook-form";
+import { useAppDispatch } from "@/state/app/hooks";
+import { handleModalStatus, ModalStatus } from "@/state/features";
 
 import { Button, Input, Textarea } from "@components/common";
 
 import styles from "./Forms.module.css";
 
 const ContactForm = () => {
-  const { register, handleSubmit } = useForm();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const dispatch = useAppDispatch();
+  const formError = errors["email*"]?.type || errors["message*"]?.type;
```

Comment:
Are you using this only to handle if there is some error but you don’t need to know which input has the error right? You could do something like
` const isFormValid = Object.keys(errors).length === 0;`

Reply:
It looks cleaner, thanks! I'll change it